data/scripts/globalevents/account_storage.lua

```diff
@@ -0,0 +1,63 @@
+-- load accounts storages
+local load = GlobalEvent("LoadAccountStorage")
+
+function load.onStartup()
+	local resultId = db.asyncStoreQuery("SELECT `account_id`, `key`, `value` FROM `account_storage`")
```

Comment:
Any problems with being async? 🦆

Reply:
you should leave it as it has always been

This change will possibly mean that when you want to verify a storage value in the onLogin event we will not be able to obtain the expected values ​​because it is likely that the asynchronous task of executing the query has not yet been executed.

Also this script is wrong, because this task doesn't really return a results ID, in fact it doesn't return anything...
An asynchronous task is supposed to have a callback that will be called when the query is executed, this callback is passed as the second parameter

```lua
db.asyncStoreQuery("SELECT `account_id`, `key`, `value` FROM `account_storage`", function(result) ... end)
```

Reply:
> you should leave it as it has always been
> 
> This change will possibly mean that when you want to verify a storage value in the onLogin event we will not be able to obtain the expected values ​​because it is likely that the asynchronous task of executing the query has not yet been executed.
> 
> Also this script is wrong, because this task doesn't really return a results ID, in fact it doesn't return anything... An asynchronous task is supposed to have a callback that will be called when the query is executed, this callback is passed as the second parameter
> 
> ```lua
> db.asyncStoreQuery("SELECT `account_id`, `key`, `value` FROM `account_storage`", function(result) ... end)
> ```

Right, I didn't notice that the callback would be a func. Added comments in luascript to help others